app/controllers/v1/pets_for_adoption_controller.rb

```diff
@@ -2,8 +2,14 @@ class V1::PetsForAdoptionController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def index
+    pets = Pet.active
+    pets = pets.by_sex(params[:sex]) if params[:sex].present?
+    pets = pets.by_description(params[:description]) if params[:description].present?
```

Comment:
I think it would be nice to include names here too.

Reply:
Good idea. I changed the description filter to consider name as well.